src/style/global-style.js

```diff
@@ -34,6 +34,33 @@ button {
   50% { opacity: 0.6 }
   to { opacity: 0.85 }
 }
+
+a {
+  border-left: 1px solid #dddddd;
+}
+
+*:focus {
+  /* Provide a fallback style for browsers
+    that don't support :focus-visible */
+  outline: 2px solid red;
+  background: transparent;
+}
```

Comment:
Hu, today I learned that `:focus-visible` is not that old yet and that Safari only shipped support a month ago, crazy. However, why would this fallback not use the same styling as `:focus-visible` below? :thinking: